class-wporg-webauthn-provider.php

```diff
@@ -0,0 +1,131 @@
+<?php
+namespace WordPressdotorg\Two_Factor;
+use TwoFactor_Provider_WebAuthn, Two_Factor_Core;
+
+/**
+ * Extends the TwoFactor_Provider_WebAuthn class for WordPress.org needs.
+ */
+class WPORG_TwoFactor_Provider_WebAuthn extends TwoFactor_Provider_WebAuthn {
+	/**
+	 * Use the parent class as the "key" in the Two Factor UI.
+	 */
+	public function get_key() {
+		return parent::class;
+	}
+
+	/**
+	 * Override the parent constructor to allow for adding filters early.
+	 *
+	 * This would be ideal as a `__construct()` method, but the parent is final and $instance is private.
+	 */
+	public static function get_instance() {
+		static $instance = false;
+		if ( ! $instance ) {
+			$instance = new static();
+		}
+
+		// Add the custom filters needed for this class
+		static $filters_added = false;
+		if ( ! $filters_added ) {
+			$filters_added = true;
+
+			$instance->_add_filters();
+		}
+
+		return $instance;
+	}
+
+	/**
+	 * Check if the provider is available for the given user.
+	 *
+	 * This method includes caching for WordPress.org, as it's called on most pageloads.
+	 */
+	public function is_available_for_user( $user ) {
+		$is_available = wp_cache_get( 'webauthn:' . $user->ID, 'users', false, $found );
+		if ( $found ) {
+			return $is_available;
+		}
+
+		$is_available = parent::is_available_for_user( $user );
+
+		wp_cache_set( 'webauthn:' . $user->ID, $is_available, 'users', HOUR_IN_SECONDS );
+
+		return $is_available;
+	}
+
+	/**
+	 * See https://github.com/sjinks/wp-two-factor-provider-webauthn/pull/468
+	 *
+	 * @return string
+	 */
+	public function get_alternative_provider_label() {
+		return __( 'Use your security key', 'wporg-two-factor' );
+	}
+
```

Comment:
```suggestion
```

https://github.com/sjinks/wp-two-factor-provider-webauthn/pull/468 has merged

Reply:
Removing this makes sense to me, but depending on how long until the next upstream release happens it might be worth just keeping this to reduce UI feedback.

Reply:
That WFM 👍🏻